src/builtin.c

```diff
@@ -1390,10 +1409,10 @@ static block bind_bytecoded_builtins(block b) {
                                             BLOCK(gen_param("start"), gen_param("end")),
                                             range));
   }
-
-  return block_bind_referenced(builtins, b, OP_IS_CALL_PSEUDO);
+  return block_bind(builtins, b, OP_IS_CALL_PSEUDO);
 }
 
+
 #define LIBM_DD(name) "def " #name ": _" #name ";"
```

Comment:
They are harmless now, but 1416-1421 should now be removed.  The comment at 1427 should also be removed.